src/coreclr/System.Private.CoreLib/src/System/Runtime/CompilerServices/RuntimeHelpers.CoreCLR.cs

```diff
@@ -139,8 +139,32 @@ public static unsafe void PrepareMethod(RuntimeMethodHandle method, RuntimeTypeH
         [MethodImpl(MethodImplOptions.InternalCall)]
         internal static extern int TryGetHashCode(object o);
 
+        public static new unsafe bool Equals(object? o1, object? o2)
```

Comment:
We do not seem to have any test coverage for this method. Could you please add some?

Reply:
I'm not sure how to express the mentioned issue about padding bits.

Reply:
If we had any regular test coverage for this API, we would likely see the tests fail intermittently due to the padding issue. 

I do not think you need to bother with adding a test that fails deterministically due the padding issue.